controllers/suite_test.go

```diff
@@ -116,7 +116,8 @@ var _ = BeforeSuite(func() {
 	By("Bootstrapping test environment")
 	envTest = &envtest.Environment{
 		CRDInstallOptions: envtest.CRDInstallOptions{
-			Paths:              []string{filepath.Join("..", "config", "crd", "external")},
+			Paths: []string{filepath.Join("..", "config", "crd", "external"),
+				filepath.Join("..", "config", "crd", "bases")},
```

Comment:
tbh, i would think this new CRD to be shipped out by kserve as an overlay not by odh-model-controller.
say, kserve is disabled, modelmesh is enabled, CRD would still be installed in the cluster, should it? 

or not include "crd/bases" from "base/kustomization.yaml". 
instead, in Operator to create it on kserve demand.

Reply:
@etirelli @israel-hdez @lburgazzoli @tomerFI @jooho @mpaulgreen 
Any thoughts on the suggestions?

Reply:
Here are some legacy comments that can drive the decision. Please feel free to override me. https://issues.redhat.com/projects/NVPE/issues/NVPE-4?filter=allissues and from a slack thread - https://redhat-internal.slack.com/archives/C07CZDA6DE1/p1730391034630149?thread_ts=1730389624.036899&cid=C07CZDA6DE1

Reply:
i do not have objection to place CRD in such a way

Reply:
No strong preference on this. I think it's fine keep it as is.

Reply:
I'm OK with having the CRD in this repo.